---
permalink: /about/
title: "About Me"
---

With a deep foundation in molecular biology and nearly a decade of integrating software development into scientific workflows, I bring expertise in experimental design, bioinformatics, data stewardship, and laboratory automation.

After earning my Ph.D. in Molecular Biology at Princeton University and completing a postdoc at the [Joint BioEnergy Institute](https://www.jbei.org), I joined the agricultural biotech company [Pivot Bio](https://www.pivotbio.com). There, I led cross-functional projects that bridged molecular biology, software development, and data infrastructure to improve research and product pipeline efficiency.

At Pivot Bio, I developed and implemented key analytical methods for evaluating microbial performance in plants. I established a next-generation sequencing (NGS)-based amplicon sequencing assay for assessing microbial colonization and activity in planta, as well as co-led the development of a novel spectroscopic analytical method for product candidate evaluation. These efforts required significant programming, data analysis, and scientific design, pushing me into the strange new world of plant biology while maintaining a rigorous experimental and computational approach. I also supported strain characterization efforts, establishing microbial naming conventions, analyzing sequencing data, and ensuring accurate strain documentation for field trials. Once these methods were well-validated, I transitioned them to other teams and took on new challenges in data infrastructure, LIMS, and workflow automation.

## Skills

- **Biotechnology**: High-throughput analytical & NGS assay development; cultivation, isolation, and genome editing of diverse strains of bacteria & yeast
- **Data Stewardship & LIMS**: CDesigning relational data models, integrating lab automation, SQL-based dashboards, workflow automation (Benchling, custom LIMS, Databricks)
- **Software Developmen & Automation**: Python-based data processing pipelines, lab instrument control, API-driven data integrations
- **Programming & Tools**: Python (Pandas, NumPy, Biopython, scikit-learn, Tkinter, Flask, Dash), Nextflow, SQL, Unix/Linux, Git, Bash, AWS

## Education

- **Ph.D. in Molecular Biology** - Princeton University
- **B.S. in Chemistry** - The University of Chicago

## Recent Publications

1. **DA Higgins**, JM Gladden, JA Kimbrel, BA Simmons, SW Singer, & MP Thelen. Guanidine Riboswitch-Regulated Efflux Transporters Protect Bacteria against Ionic Liquid Toxicity. Journal of Bacteriology. 201, e00069-19 (2019) [https://doi.org/10.1128/jb.00069-19]

2. **DA Higgins**, MKM Young, M Tremaine, M Sardi, JM Fletcher, M Agnew, L Liu, Q Dickinson, D Peris, RL Wrobel, CT Hittinger, AP Gasch, SW Singer, BA Simmons, R Landick, MP Thelen, & TK Sato. Natural Variation in the Multidrug Efflux Pump SGE1 Underlies Ionic Liquid Tolerance in Yeast. Genetics. 210, 219-234 (2018) [https://doi.org/10.1534/genetics.118.301161]

## Contact

Feel free to [contact me](/contact/) if you're interested in collaborating or learning more about my work.
